Dockerfile

```diff
@@ -3,11 +3,13 @@ WORKDIR /app
 COPY requirements.txt .
 RUN pip install --no-cache-dir -r requirements.txt
 
-FROM base AS lint
-RUN pip install --no-cache-dir pylint
-COPY *.py ./
+FROM base AS dev
+# Alas, black depends on regex, which doesn't have an Alpine wheel, which means
+# we need to compile it, which means we need a build environment.
+RUN apk add --no-cache --virtual .black-build-deps gcc musl-dev \
+   && pip install --prefer-binary --no-cache-dir pylint black isort \
+   && apk del --no-cache .black-build-deps
```

Comment:
Thank you!
I feel like the only appropriate response to this level of headache is to award you one Austrian Yak for your troubles:
![Yak](https://upload.wikimedia.org/wikipedia/commons/thumb/3/32/Yak_Kailash.JPG/273px-Yak_Kailash.JPG)

Reply:
I really do think it's the right way to go, the errors were clear, and the internet had my back.